lua/nvim-treesitter/parsers.lua

```diff
@@ -308,6 +308,14 @@ list.scss = {
   maintainers = { "@elianiva" },
 }
 
+list.sql = {
+  install_info = {
+    url = "https://github.com/m-novikov/tree-sitter-sql",
+    files = { "src/parser.c" }
+  },
+  maintainers = { "@m-novikov" }
```

Comment:
```suggestion
  maintainers = { "@m-novikov" },
```

Reply:
And @m-novikov has volunteered to maintain the parser (and queries) _on `nvim-treesitter`_?

Reply:
Thanks for bringing this to my attention. 
I am nvim-treesitter user and was planning to making same PR at some point. But I am not sure that my implementation of the parser currently covers all use cases.
I am going to dump some more time in this project in next couple of weeks.

Reply:
thanks @m-novikov, I'll make these linter changes but I assume that this pr (or one like it) is probably better in your hands. @clason, is there some sort of submodule system, or ci system that nvim-treesitter plans to use to make slightly more seamless management by outside maintainers? The whole copy/paste system right now feels a bit disjointed

Reply:
What copy/paste system?

Queries are not (entirely) editor-agnostic, so we almost never take upstream queries unchanged.

Reply:
would be open to auto-import queries from repos that explicitly maintain nvim-treesitter compatible queries and have them reviewed in similar PRs like the reviewed in the update-parser updates.

Reply:
Would have to be atomic with the lockfile updates, though, otherwise they could end up blocking each other.

Reply:
Note that there are also highlight test cases that should be imported from the repo